ios/sdk/WeexSDK/Sources/Layout/WXComponent+Layout.m

```diff
@@ -205,6 +205,10 @@ - (void)_layoutDidFinish
         [self.ancestorScroller adjustSticky];
     }
     
+    if (0 == [[self.view subviews] count]) {
+        self.view.isAccessibilityElement = YES;
+        self.view.accessibilityFrame = [[self.view superview] convertRect:self.calculatedFrame toView:nil];
+    }
```

Comment:
You do not have to set this property for an accessibility element that represents a subclass of UIView, because such an object’s screen coordinates are already known.